src/main/java/org/broadinstitute/hellbender/utils/GATKProtectedVariantContextUtils.java

```diff
@@ -263,4 +273,21 @@ public static void setGenotypeQualityFromPLs(final GenotypeBuilder builder, fina
             return translate.apply(String.valueOf(value));
         }
     }
+
+    /**
+     * Get the pileup of reads covering a locus.  This is useful, for example, in VariantWalkers, which work on
+     * ReadsContexts and not AlignmentContexts.
+     */
+    public static ReadPileup getPileup(final Locatable loc, Iterable<GATKRead> reads) {
+        final List<PileupElement> pile = StreamSupport.stream(reads.spliterator(), false)
+                .filter(r -> !r.failsVendorQualityCheck())
```

Comment:
You might want to use static Predicates defined in `ReadFilterLibrary.java` like `ReadFilterLibrary.PassesVendorQualityCheckReadFilter`. They do the same thing, it's just a matter of style.